.jenkins/infrastructure/build_base_azure_gallery_image.Jenkinsfile

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+
+library "OpenEnclaveJenkinsLibraryCyan@${params.OECI_LIB_VERSION}"
+
+def buildLinuxVMBaseImage(String os_type, String os_version) {
+    stage('Environment setup') {
+        withCredentials([usernamePassword(credentialsId: 'SERVICE_PRINCIPAL_OSTCLAB',
+                                        passwordVariable: 'SERVICE_PRINCIPAL_PASSWORD',
+                                        usernameVariable: 'SERVICE_PRINCIPAL_ID'),
+                        string(credentialsId: 'OSCTLabSubID', variable: 'SUBSCRIPTION_ID'),
+                        string(credentialsId: 'TenantID', variable: 'TENANT_ID'),
+        ]) {
+            retry(10) {
+                sh """
+                    sleep 5
```

Comment:
Doesn't `WaitForAptLock()` already have a sleep? If so, wouldn't this line be redundant?

Reply:
Yes `WaitForAptLock()` does have a sleep time, but only sleeps if there is a lock or apt/dpkg process. Otherwise there is no sleep. So `sleep 5` here provides a guaranteed buffer time between retries.